resources/lang/en/thoughts/alive/general.json

```
@@ -1325,7 +1325,7 @@
             "Thinks that {PRONOUN/m_c/subject} could take on a dog solo",
             "Hopes to earn more battle scars to show off",
             "Wishes {PRONOUN/m_c/subject} had more scars",
-            "Pounced on a Clanmate unexpectedly, meowing about how they should be ready at any moment",
+            "Pounces on r_c and says {PRONOUN/r_c/subject} should always be vigilant",
```

```suggestion
            "Pounces on r_c unexpectedly, saying that {PRONOUN/r_c/subject} should always be vigilant",
```
i prefer this one with the word unexpectedly, but that's a bit of a purrsonal purrefurence :p

---

Yeahh I went back and forth with it -- imo the unexpectedness of the pounce is implied by "you should always be vigilant". The other reason is that the word "unexpectedly" is the reason this thought is so long.

---

If you have any thoughts on how to maintain that feeling while keeping the thought short, pls do share though :')